test/org/loklak/api/search/GithubProfileScraperTest.java

```diff
@@ -0,0 +1,56 @@
+package org.loklak.api.search;
+
+import org.junit.Test;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.loklak.api.search.GithubProfileScraper;
+import org.loklak.susi.SusiThought;
+
+public class GithubProfileScraperTest {
+	@Test
+	public void githubProfileScraperOrgTest() {
+		String profile = "fossasia";
+		String shortDescription = "Open Technologies in Asia";
+		String userName = "fossasia";
+		String userId = "6295529";
+		String location = "Singapore";
+		String specialLink = " http://fossasia.org";
+
+		SusiThought response = GithubProfileScraper.scrapeGithub(profile);
+		JSONObject fetchedProfile = (JSONObject)response.getData().get(0);
+
+		assertTrue(fetchedProfile.has("short_description"));
+		assertThat(fetchedProfile.getString("short_description"), is(shortDescription));
+		assertTrue(fetchedProfile.has("user_name"));
+		assertThat(fetchedProfile.getString("user_name"), is(userName));
+		assertTrue(fetchedProfile.has("user_id"));
+		assertThat(fetchedProfile.getString("user_id"), is(userId));
+		assertTrue(fetchedProfile.has("location"));
+		assertThat(fetchedProfile.getString("location"), is(location));
+		assertTrue(fetchedProfile.has("special_link"));
```

Comment:
hey , nice job, but please use assertEquals for strings, it gives better message for strings.